package.json

```diff
@@ -21,9 +22,11 @@
     "formik-material-ui": "^2.0.0-beta.1",
     "graphql": "^15.0.0",
     "react": "^16.13.1",
+    "react-data-table-component": "^6.9.3",
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.1",
+    "styled-components": "^5.1.1",
```

Comment:
Currently, our plan is to have CSS in separate files only. We should not add 'styled-components' package. The only exception is when we need to reuse some of the functions of them then we can add CSS in components.

Reply:
Looks like this is a dependency for react data table